.github/CODEOWNERS

```diff
@@ -31,3 +30,7 @@ rust_g.dll @AffectedArc07
 
 # TGUI stuff
 /tgui/bin @S34NW
+
+
```

Comment:
```suggestion
```

Reply:
Theyre double spaced between people

Reply:
Ah rest all had single spaces in the file, feel free to resolve